contrib/depends/packages/sodium.mk

```diff
@@ -15,6 +15,7 @@ endef
 define $(package)_config_cmds
   ./autogen.sh &&\
   patch -p1 < $($(package)_patch_dir)/fix-whitespace.patch &&\
+  patch -p1 < $($(package)_patch_dir)/disable-glibc-getrandom-getentropy.patch &&\
```

Comment:
You need to include the patch in the `$(package)_patches` section above.

Reply:
Done

Reply:
The patch needs to be before autogen, otherwise it does not reach the configure script. So move it one line up (but keep the other patch where it is).

Reply:
Yeah